distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/FederatedCswServer.java

```diff
@@ -0,0 +1,205 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.common.test.restito;
```

Comment:
The comment at the top of that class actually mentions that it's a workaround for a commit that @ryeats contributed to restito, which should be in version 0.8.2. Maybe we should just upgrade to that version instead?